node/narwhal/src/helpers/dag.rs

```diff
@@ -101,7 +101,8 @@ impl<N: Network> DAG<N> {
         let author = certificate.author();
 
         // Update the last committed round for the author.
-        self.last_committed_authors
+        let last_committed_author = self
```

Comment:
nit: this should read `last_committed_round` (the type is a round, not an author)